### ISSUE TITLE
create release build with debug info

### ISSUE THREAD

**Issue by sureshyerva87** [microsoft/ChakraCore#6353]
I am using 1.11.12 version of chakracore (on Mac). Using build.sh, I've created release build. But, how to create release build with debug info for chakraCore?

**Comment by rhuanjl** [comment]
Unless you're using undocumented features the Test build and release build should behave the same way except for the release build not having the debug symbols.

The test build exposes some extra config options BUT the normal API doesn't actually give you access to them so it should be irrelevant.

If you share your code I could try and work out what may be causing the crash.

**Comment by rhuanjl** [comment]
I don't think we want to add symbols to a standard release build, and to debug this specific crash more info is needed - will await extra information.
